### DISCUSSION TITLE
Does signoz support alerting ?

### DISCUSSION THREAD

**Discussion by kholisrag** [SigNoz/signoz#197] [Q&A] [answered]
Hi,
Quick question,
Does signoz support alerting?

**Comment by pranay01** [comment]
Update Dec 6, 2021 - 

We have released support for alerts with [v0.5.0](https://github.com/SigNoz/signoz/releases/tag/v0.5.0)

Would love it if we you can check it out and share any feedback. Here are the [docs](https://signoz.io/docs/userguide/alerts-management/) for the same

----

Hey, no we don't support alerting today. But we will be bringing it soon.

Do you want to open an issue for it with your alerting use case and what you would like to see? Currently our plan is to build a promql like interface for alerting

**Reply by kholisrag** [reply]
well that's good actually, maybe with scraping custom metrics too, like what prometheus do 😅

**Reply by pranay01** [reply]
Yup. Curious, what alerting tool do you use currently for your monitoring stack?

**Reply by kholisrag** [reply]
prometheus -> alertmanager -> opsgenie -> slack

**Reply by mrmarcsmith** [reply]
I would love to see this feature added too! we currently use Datadog > Monitoring > Slack AND OpsGenie AND Email

**Reply by pranay01** [reply]
hey @mrmarcsmith Thanks for your note. We have already started working on support for alert in SigNoz. Should be open it for beta testing in a months time

**Reply by kholisrag** [reply]
Any news?

**Reply by pranay01** [reply]
hey @petrukngantuk We are building support for alerts in current sprint - should be available for beta testing in 2 weeks? Would you like to be one of the beta testers? If yes, can you let me know your email id or ping me on our slack community [here](https://bit.ly/signoz-slack)

**Reply by pranay01** [reply]
@kholisrag @mrmarcsmith We have released support for alerts with [v0.5.0](https://github.com/SigNoz/signoz/releases/tag/v0.5.0)

Would love it if we you can check it out and share any feedback. Here are the [docs](https://signoz.io/docs/userguide/alerts-management/) for the same

**Comment by pranay01** [comment, marked as answer]
We have released support for alerts with [v0.5.0](https://github.com/SigNoz/signoz/releases/tag/v0.5.0)

Would love it if we you can check it out and share any feedback. Here are the [docs](https://signoz.io/docs/userguide/alerts-management/) for the same
